cartography/intel/gcp/compute.py

```diff
@@ -114,6 +114,24 @@ def get_gcp_instance_responses(project_id: str, zones: Optional[List[Dict]], com
     for zone in zones:
         req = compute.instances().list(project=project_id, zone=zone['name'])
         res = req.execute()
+        for item in res.get('items',[]):
+            public_facing = []
+            for networkinterface in item.get('networkInterfaces',[]):
+                for accessconfig in networkinterface.get('accessConfig',[]):
+                    public_facing.append(accessconfig.get('name',None))
+            item['accessConfig'] = public_facing
+            item['iam_policy'] = compute.instances().getIamPolicy(\
+                project=project_id,zone=zone['name'],resource=item.get("id")).execute()
+            bindings = item.get('iam_policy',{}).get('bindings',[])
+            members = []
+            for binding in bindings:
+                members.append(binding.get('members'))
+            item['members'] = members
+            for member in item['members']:
+                if member.startswith('allUsers'):
+                    item['user'] = 'allUsers'
+                elif member.startswith('allAuthenticatedUsers'):
+                    item['user'] = 'allAUthenticatedUsers'
```

Comment:
did we miss to add the relationship between the iam entities and compute instance? So that we can use that for the public facing calculation.